### PR TITLE
Header

### PR DESCRIPTION
**Pull request by olenakorol1708**
Added a branch for  header components. I took into account the advice regarding the include-media library, shortened the paths to the components by adding index.ts , added a calculation formula for converting from pх to rem, and laid out a header component with an adaptive layout for a tablet.

### CODE REVIEW

File: src/Components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import logoImage from "../../images/XriiLogo/gradient logo.svg";
+import { Link } from "react-router-dom";
+import paths from "../../Paths/paths";
```
Comment (lluneva) [inline src/Components/Header/Header.tsx:6]: you can use alias @/ also here

File: src/Components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <Link to={paths.HOME}>
+          <img src={logoImage} alt="companyLogo" />
+        </Link>
+      </div>
+      <nav className="header_navigation">
+        <Link to={paths.CASE}>Case studies</Link>
+        <Link to={paths.PRICE}>Pricing</Link>
+          <Link to={paths.ABOUT}>About</Link>
+        <Link to={paths.BLOG}>Blog</Link>
+        <Link to={paths.HOME}></Link>
```
Comment (lluneva) [inline src/Components/Header/Header.tsx:31]: I wonder if this last one is needed, as the first one -logo is Home. And this one does not even have any content/ children

File: src/images/XriiLogo/gradient logo.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<svg width="157" height="72" viewBox="0 0 157 72" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M81.0234 31.0267L91.6631 15.8461C91.8602 15.5503 91.8602 15.3532 91.8602 15.2546C91.8602 14.9589 91.5647 14.7617 91.0721 14.7617H89.6928C89.2987 14.7617 89.0032 14.8603 88.8062 14.9589C88.6091 15.0574 88.3136 15.3532 88.018 15.7475L79.2501 28.4637L70.4821 15.7475C70.1865 15.2546 69.891 15.0574 69.6939 14.9589C69.4969 14.8603 69.2014 14.7617 68.8073 14.7617H67.3295C66.837 14.7617 66.5414 14.9589 66.5414 15.2546C66.5414 15.3532 66.6399 15.5503 66.7384 15.8461L77.3782 31.1253L66.2459 47.0945C66.0488 47.3902 66.0488 47.5874 66.0488 47.686C66.0488 47.9817 66.3444 48.1788 66.837 48.1788H68.2162C68.6103 48.1788 68.9058 48.0803 69.1028 47.9817C69.2999 47.8831 69.5954 47.5874 69.891 47.1931L79.1515 33.7868L88.4121 47.1931C88.7077 47.686 89.0032 47.8831 89.2002 47.9817C89.3973 48.0803 89.6928 48.1788 90.0869 48.1788H91.5647C92.0572 48.1788 92.3528 47.9817 92.3528 47.686C92.3528 47.5874 92.2542 47.3902 92.1557 47.0945L81.0234 31.0267Z" fill="white"/>
```
Comment (lluneva) [inline src/images/XriiLogo/gradient logo.svg:2]: check file name - it should not have spaces in filenames

File: src/index.scss [hunk 4]
```diff
@@ -6,18 +6,32 @@
...
+*::after{
+  box-sizing: border-box;
+}
 
 body {
-  margin: 0;
-  display: flex;
-  place-items: center;
+  
+  // display: flex;
```
Comment (lluneva) [inline src/index.scss]: the best practice is not to have commented lines in code

File: src/index.scss [hunk 5]
```diff
@@ -6,18 +6,32 @@
...
+  // place-items: center;
   min-width: 320px;
   min-height: 100vh;
   background-color: $background-color;
   
 }
-
+.container
+{
```
Comment (lluneva) [inline src/index.scss]: Hmm, interesting, that the bracket is on next line. I wonder if you have code formatter  (like prettier),running on this file, so it is auto formated

**Review by lluneva — APPROVED** [review]

File: src/Components/Header/Header.tsx (hunk 1 above)
Comment (olenakorol1708) [reply src/Components/Header/Header.tsx:6]: Fixed to import paths from '@paths/paths';

File: src/Components/Header/Header.tsx (hunk 2 above)
Comment (olenakorol1708) [reply src/Components/Header/Header.tsx:31]: Changed to <Link to={paths.HOME}>Home</Link>. Added  link "HOME" to navigation.

File: src/images/XriiLogo/gradient logo.svg (hunk 3 above)
Comment (olenakorol1708) [reply src/images/XriiLogo/gradient logo.svg:2]: Changed to gradient_logo.

File: src/index.scss (hunk 4 above)
Comment (olenakorol1708) [reply src/index.scss]: Fixed.

File: src/index.scss (hunk 5 above)
Comment (olenakorol1708) [reply src/index.scss]: my prettier doesn't work automatically.try to fix it

**Pull request merged**